<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBook</class>
 <widget class="QMainWindow" name="CreateBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
     <string>MainWindow</string>
  </property>
 <widget class="QWidget" name="centralwidget">
  <widget class="QLineEdit" name="authorLineEdit">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>270</y>
     <width>340</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>270</y>
     <width>194</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>193</y>
     <width>194</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleLineEdit">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>193</y>
     <width>340</width>
     <height>39</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>891</x>
     <y>49</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons8-return-64.png</normaloff>
     <normalon>:/images/images/icons8-return-64.png</normalon>:/images/images/icons8-return-64.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>64</y>
     <width>451</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create a new book</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveAccountButton">
   <property name="geometry">
    <rect>
     <x>891</x>
     <y>611</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Historic</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/icons8-save-close-64.png</normaloff>:/images/images/icons8-save-close-64.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="img_smallLineEdit">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>347</y>
     <width>340</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="img_fullLineEdit">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>420</y>
     <width>340</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="smallImageURLLabel">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>347</y>
     <width>194</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Thumbnail URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="fullImageURLLabel">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>424</y>
     <width>194</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Full Image URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>647</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(226, 190, 243)</string>
   </property>
   <property name="text">
    <string>Book Created</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
 </widget>
 <widget class="QMenuBar" name="menubar">
<property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
